### PR TITLE
docs: add sp1 and risc0 versions

### LINKED ISSUE #563: docs: add versions of sp1 and risc0
# Description

- We are missing the versions of sp1 and risc0 in the docs.

### PR DESCRIPTION
**Pull request by NicolasRampoldi**
# Description

- This PR adds sp1 and risc0 versions to the docs.

### CODE REVIEW

**Review by entropidelic — APPROVED** [review]

File: docs/architecture/0_supported_verifiers.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ The following is the list of the verifiers currently supported by Aligned:
 
 - :white_check_mark: gnark - Groth16 (with BN254)
 - :white_check_mark: gnark - Plonk (with BN254 and BLS12-381)
-- :white_check_mark: SP1
-- :white_check_mark: Risc0
+- :white_check_mark: SP1 (v1.0.8-testnet)
```
Comment (JuArce) [inline docs/architecture/0_supported_verifiers.md]: Maybe we should add a link to the release

File: docs/architecture/0_supported_verifiers.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ The following is the list of the verifiers currently supported by Aligned:
...
+- :white_check_mark: Risc0 (v1.0.1)
```
Comment (JuArce) [inline docs/architecture/0_supported_verifiers.md]: Maybe we should add a link to the release

File: docs/guides/0_submitting_proofs.md [hunk 3]
```diff
@@ -10,8 +10,8 @@ The following is the list of the verifiers currently supported by Aligned:
 
 - :white_check_mark: gnark - Groth16 (with BN254)
 - :white_check_mark: gnark - Plonk (with BN254 and BLS12-381)
-- :white_check_mark: SP1
-- :white_check_mark: Risc0
+- :white_check_mark: SP1 (v1.0.8-testnet)
```
Comment (JuArce) [inline docs/guides/0_submitting_proofs.md]: Maybe we should add a link to the release

File: docs/guides/0_submitting_proofs.md [hunk 4]
```diff
@@ -10,8 +10,8 @@ The following is the list of the verifiers currently supported by Aligned:
...
+- :white_check_mark: Risc0 (v1.0.1)
```
Comment (JuArce) [inline docs/guides/0_submitting_proofs.md]: Maybe we should add a link to the release

File: docs/operator_guides/0_running_an_operator.md [hunk 5]
```diff
@@ -9,8 +9,8 @@ The following is the list of the verifiers currently supported by Aligned:
 
 - :white_check_mark: gnark - Groth16 (with BN254)
 - :white_check_mark: gnark - Plonk (with BN254 and BLS12-381)
-- :white_check_mark: SP1
-- :white_check_mark: Risc0
+- :white_check_mark: SP1 (v1.0.8-testnet)
```
Comment (JuArce) [inline docs/operator_guides/0_running_an_operator.md]: Maybe we should add a link to the release

File: docs/operator_guides/0_running_an_operator.md [hunk 6]
```diff
@@ -9,8 +9,8 @@ The following is the list of the verifiers currently supported by Aligned:
...
+- :white_check_mark: Risc0 (v1.0.1)
```
Comment (JuArce) [inline docs/operator_guides/0_running_an_operator.md]: Maybe we should add a link to the release

**Review by JuArce — CHANGES_REQUESTED** [review]
I think we are putting the version everywhere. Maybe it is not necessary

File: docs/operator_guides/0_running_an_operator.md (hunk 6 above)
Comment (NicolasRampoldi) [reply docs/operator_guides/0_running_an_operator.md]: Fixed in [3f314d7](https://github.com/yetanotherco/aligned_layer/pull/566/commits/3f314d724c1dd1e7e461a1d2f1d47ef5520c27fc)

**Review by JuArce — APPROVED** [review]

**Pull request merged**